<launch>
  <arg name="goal_topic_name" default="/move_base_simple/goal" />
  <arg name="checkpoint_topic_name" default="/move_base_simple/goal" />
  <arg name="rout_topic_name" default="/planning/mission_planning/route" />
  <arg name="map_topic_name" default="/vector_map" />
  <arg name="visualization_topic_name" default="/planning/mission_planning/route_marker" />

  <node pkg="mission_planner" type="mission_planner" name="mission_planner" output="screen">
    <param name="map_frame" value="map" />
    <param name="base_link_frame" value="base_link" />
    <remap from="~input/vector_map" to="$(arg map_topic_name)" />
    <remap from="~input/goal_pose" to="$(arg goal_topic_name)" />
    <remap from="~input/checkpoint" to="$(arg checkpoint_topic_name)" />
    <remap from="~output/route" to="$(arg rout_topic_name)" />
    <remap from="~debug/route_marker" to="$(arg visualization_topic_name)" />
  </node>
</launch>